internal/store/postgres/migrations/000009_add_grant_source_column.up.sql

```diff
@@ -0,0 +1,22 @@
+BEGIN;
+
+ALTER TABLE
+  "grants"
+ADD
+  COLUMN IF NOT EXISTS "source" text,
+ADD
+  COLUMN IF NOT EXISTS "status_in_provider" text,
+ADD
+  COLUMN IF NOT EXISTS "owner" text;
+
+UPDATE
+  "grants"
+SET
+  "source" = 'appeal',
+  "status_in_provider" = "status",
+  "owner" = "created_by";
```

Comment:
same with this, is this for renaming column or setting the specific column with specific value?

Reply:
yes renaming column, and setting `"grants"."owner"` to `"grants"."created_by"`

Reply:
And for the `"source" = 'appeal',`, is the single quote `'` there is mandatory or could we make it the same with the others and use `"` double quotes?

Reply:
single quotes are for string/text value, so the value is going to be exactly "appeal". And the double quotes are for referring to the column name, we are setting the value of `"grants"."owner"` to be the current value of `"grants"."created_by"`